.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Release Artifacts
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 14
+    - name: Checkout Data-Prepper
+      uses: actions/checkout@v2
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build
+      run: ./gradlew build
+    - name: Archives
+      run: ./gradlew :release:archives:linux:linuxTar -Prelease
+    - name: Docker
```

Comment:
Take it or leave it: I would recommend making the names more explicit as `Build`, `Archives`, and `Docker` are a little ambiguous. I like the names you provided in on of the diagrams in the RFC:

- Build jar files
- Build archives || Build tar.gz files
- Build docker image

Reply:
Thanks! I'll update this in the next PR.